src/SMAIAXBackend.API/Endpoints/Order/OrderSmartMeterConnectorEndpoint.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.AspNetCore.Http.HttpResults;
+using Microsoft.AspNetCore.Mvc;
+
+using SMAIAXBackend.Application.Services.Interfaces;
+
+namespace SMAIAXBackend.API.Endpoints.Order;
+
+public static class OrderSmartMeterConnectorEndpoint
+{
+    public static async Task<Ok<Guid>> Handle(
+        IOrderService orderService,
+        IKeyGenerationService keyGenerationService)
+    {
+        var connectorSerialNumber = await orderService.OrderSmartMeterConnectorAsync();
```

Comment:
Wouldn't it be nicer to return a DTO? Now you don't really know the GUID of what is returned here.